app/brave_settings_strings.grdp

```diff
@@ -1159,6 +1159,18 @@
   <message name="IDS_SETTINGS_WALLET_RESET_CONFIRMATION_PHRASE" desc="The phrase users should type to reset wallet">
     Yes
   </message>
+  <message name="IDS_BRAVE_WALLET_TRANSACTION_SIMULATIONS_DESC" desc="The description for the user's opt-in status of the transaction simulations service">
+    See the results of transactions before you sign
+  </message>
+  <message name="IDS_BRAVE_WALLET_TRANSACTION_SIMULATIONS_OPT_IN_ENBABLED" desc="Select control value for opting in or out of transaction simulations">
```

Comment:
For Yes and Ask, how about we just reuse existing translations in this file?

Reply:
updated: aded764